pkgs/applications/networking/irc/weechat/scripts/weechat-matrix/default.nix

```diff
@@ -20,15 +20,17 @@ let
     requests
     python_magic
   ]);
+
+  version = "0.2.0";
 in buildPythonPackage {
   pname = "weechat-matrix";
-  version = "0.1.0";
+  inherit version;
 
   src = fetchFromGitHub {
     owner = "poljar";
     repo = "weechat-matrix";
-    rev = "65a5db7291439b6132e35e8cc09ed901614fabf6";
-    sha256 = "0m3k5vrv5ab1aw1mjd0r8d71anwqzvncvv9v5zx9xp1i188sdm8x";
+    rev = version;
+    hash = "sha256-qsTdF9mGHac4rPs53mgoOElcujicRNXbJ7GsoptWSGc=";
```

Comment:
Unfortunately, this syntax of setting hashes is valid only with versions of nix more recent than the current support level of nixpkgs, so unless I've missed us bumping it it's not possible to merge this as-is :/ 

When you get to changing this to a `sha256 =` hash, feel free to ping me! :)

Reply:
People on #nixos-dev IRC have told me this format is fine now.

Reply:
https://logs.nix.samueldr.com/nixos-dev/2020-09-09#3959626